tools/inliner.c

```
@@ -12,24 +12,98 @@
  * C runtime and essential libraries.
  */
 
+#include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 
 #define MAX_LINE_LEN 200
-#define MAX_SIZE 65536
+#define DEFAULT_SIZE 65536
```

The use of `DEFAULT_SIZE` is misleading. Do you have other "size" or amounts?